tests/tracer/fixtures/ci/appveyor.json

```diff
@@ -7,9 +7,9 @@
       "APPVEYOR_BUILD_NUMBER": "appveyor-pipeline-number",
       "APPVEYOR_REPO_BRANCH": "master",
       "APPVEYOR_REPO_COMMIT": "appveyor-repo-commit",
-      "APPVEYOR_REPO_COMMIT_MESSAGE_EXTENDED": "appveyor-commit-message",
       "APPVEYOR_REPO_COMMIT_AUTHOR": "appveyor-commit-author-name",
       "APPVEYOR_REPO_COMMIT_AUTHOR_EMAIL": "appveyor-commit-author-email@datadoghq.com",
+      "APPVEYOR_REPO_COMMIT_MESSAGE_EXTENDED": "appveyor-commit-message",
```

Comment:
it doesn't look like anything is changing in this file? seems like moving these keys to all be alphabetical order?

Reply:
Same with the other files, looks like some new test cases, alphabetical sorting of the keys.

The only ones that are really "different" are Gitlab CI_COMMIT_BRANCH -> CI_COMMIT REF_NAME, and Azure adding stage and job name ?

Reply:
> The only ones that are really "different" are Gitlab CI_COMMIT_BRANCH -> CI_COMMIT REF_NAME, and Azure adding stage and job name ?
that's correct. Sorry about that, we have unstable sorting in the spec repo

Reply:
👍🏻 no worries, I assume these are copy/past from a source, so as long as we know what needs review then all good